pkg/k8s/pipelineRun.go

```diff
@@ -22,6 +22,7 @@ type PipelineRun interface {
 	GetKey() string
 	GetRunNamespace() string
 	GetNamespace() string
+	GetRepoServerURL() (string, error)
```

Comment:
Not sure about the name. When I read `RepoServerURL` on a `PipelineRun` I would expect the repo where the project sources are located. What about:
```suggestion
	GetJenkinsfileRepoServerURL() (string, error)
```

Reply:
I would call it `GetPipelineRepoServerURL`.